pkg/crc/cache/cache.go

```diff
@@ -20,8 +20,8 @@ type Cache struct {
 	destDir    string
 }
 
-func New(binarName string, archiveURL string, destDir string) *Cache {
-	return &Cache{binaryName: binarName, archiveURL: archiveURL, destDir: destDir}
+func New(binaryName string, archiveURL string, destDir string) *Cache {
+	return &Cache{binaryName: binaryName, archiveURL: archiveURL, destDir: destDir}
 }
```

Comment:
nit: This typo fix is totally unrelated to these changes, only mentioning it since it took me a while to notice `binarName` vs `binaryName`

Reply:
will put this is in a separate commit